### PR TITLE
Expose virtual method in `MarkdownImage`

### PR DESCRIPTION
**Pull request by gagahpangeran**
In order to resolve (https://github.com/ppy/osu/issues/13418), need these method so I can override some parts in `MarkdownImage`

I don't have the solution for the actual issue yet, still figuring out about it.

### CODE REVIEW

**Comment by peppy** [comment]
Will wait to see how these are used before merging. You are aware of how to use a local framework checkout yeah?

**Comment by gagahpangeran** [comment]
Yeah, PR-ing this for visibility now.

**Comment by frenzibyte** [comment]
Probably worth getting a working implementation first, then start PR'ing the required parts for discussing it. (since either way, the usage of these changes is probably required for considering this and whether it should be abstracted this way etc.)

File: osu.Framework/Graphics/Containers/Markdown/MarkdownImage.cs [hunk 1]
```diff
@@ -48,20 +50,26 @@ public ImageContainer(string url)
...
+                InternalChild = image = CreateSpriteImage();
             }
 
             [BackgroundDependencyLoader]
             private void load(TextureStore textures)
+            {
+                image.Texture = GetImageTexture(textures, url);
+            }
+
+            protected virtual Sprite CreateSpriteImage() => new Sprite();
```
Comment (smoogipoo) [inline osu.Framework/Graphics/Containers/Markdown/MarkdownImage.cs:62]: I think `CreateImageSprite()` makes more sense.

**Review by smoogipoo — CHANGES_REQUESTED** [review]

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**